crates/handlers/src/admin/v1/policy_data/get_latest.rs

```diff
@@ -0,0 +1,149 @@
+            Self::NotFound => StatusCode::NOT_FOUND,
+        };
+        (status, Json(error)).into_response()
+    }
+}
+
+pub fn doc(operation: TransformOperation) -> TransformOperation {
+    operation
+        .id("getLatestPolicyData")
+        .summary("Get the latest policy data")
+        .tag("policy-data")
+        .response_with::<200, Json<SingleResponse<PolicyData>>, _>(|t| {
+            let [sample, ..] = PolicyData::samples();
+            let response = SingleResponse::new_canonical(sample);
+            t.description("Latest policy data was found")
+                .example(response)
+        })
+        .response_with::<404, RouteError, _>(|t| {
+            let response = ErrorResponse::from_error(&RouteError::NotFound);
+            t.description("No policy data was found").example(response)
+        })
+}
+
+#[tracing::instrument(name = "handler.admin.v1.policy_data.get_latest", skip_all, err)]
+pub async fn handler(
+    CallContext { mut repo, .. }: CallContext,
+) -> Result<Json<SingleResponse<PolicyData>>, RouteError> {
+    let policy_data = repo
+        .policy_data()
+        .get()
+        .await?
+        .ok_or(RouteError::NotFound)?;
+
+    Ok(Json(SingleResponse::new_canonical(policy_data.into())))
+}
+
+#[cfg(test)]
+mod tests {
+    use hyper::{Request, StatusCode};
+    use insta::assert_json_snapshot;
+    use sqlx::PgPool;
+
+    use crate::test_utils::{RequestBuilderExt, ResponseExt, TestState, setup};
+
+    #[sqlx::test(migrator = "mas_storage_pg::MIGRATOR")]
+    async fn test_get_latest(pool: PgPool) {
+        setup();
+        let mut state = TestState::from_pool(pool).await.unwrap();
+        let token = state.token_with_scope("urn:mas:admin").await;
+
+        let mut rng = state.rng();
+        let mut repo = state.repository().await.unwrap();
+
+        repo.policy_data()
+            .set(
+                &mut rng,
+                &state.clock,
+                serde_json::json!({"hello": "world"}),
+            )
+            .await
+            .unwrap();
+
+        repo.save().await.unwrap();
+
+        let request = Request::get("/api/admin/v1/policy-data/latest")
+            .bearer(&token)
+            .empty();
+        let response = state.request(request).await;
+        response.assert_status(StatusCode::OK);
+        let body: serde_json::Value = response.json();
+        assert_json_snapshot!(body, @r###"
+        {
+          "data": {
+            "type": "policy-data",
+            "id": "01FSHN9AG0MZAA6S4AF7CTV32E",
+            "attributes": {
+              "created_at": "2022-01-16T14:40:00Z",
+              "data": {
+                "hello": "world"
+              }
+            },
+            "links": {
+              "self": "/api/admin/v1/policy-data/01FSHN9AG0MZAA6S4AF7CTV32E"
+            }
+          },
+          "links": {
+            "self": "/api/admin/v1/policy-data/01FSHN9AG0MZAA6S4AF7CTV32E"
+          }
```

Comment:
the self-links are already a little verbose, but maybe fair enough. Slight eyebrow raise at it being duplicated, though

Reply:
We're following the stuff from [JSON:API](https://jsonapi.org) here, that's why

Reply:
to be fair, except for relationships the `links` field is a MAY (optional) in the spec for these cases. But sure, whatever. I don't particularly mind, I also don't really see why it's useful.